spec/controllers/faye_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe FayeController do
+  include Devise::TestHelpers
```

Comment:
I tried adding it to the spec_helper and I get and uninitialized constant error. When I add it outside config just as is only one test fails. However, if I add it using config like such:

config.include Devise::TestHelpers

I get almost all tests failing.